lib/vmpooler/metrics.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Vmpooler
+  class Metrics
+    # static class instantiate appropriate metrics object.
+    def self.init(logger, params)
+      if params[:statsd]
+        metrics = Vmpooler::Statsd.new(logger, params[:statsd])
+      elsif params[:graphite]
+        metrics = Vmpooler::Graphite.new(logger, params[:graphite])
+      elsif params[:prometheus]
+        metrics = Vmpooler::Promstats.new(logger, params[:prometheus])
+      else
+        metrics = Vmpooler::DummyStatsd.new
```

Comment:
These class names bother me... I feel like they should be more like this:
```suggestion
        metrics = Vmpooler::Metrics::Statsd.new(logger, params[:statsd])
      elsif params[:graphite]
        metrics = Vmpooler::Metrics::Graphite.new(logger, params[:graphite])
      elsif params[:prometheus]
        metrics = Vmpooler::Metrics::Promstats.new(logger, params[:prometheus])
      else
        metrics = Vmpooler::Metrics::DummyStatsd.new
```

This is because of two main reasons:

1. They are located in `vmpooler/metrics/*`
2. They are all metrics-related components.